mautic/mautic
# Prevent malformed phone numbers from causing an uncaught exception when executing campaigns

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? |  n
| Related user documentation PR URL | na
| Related developer documentation PR URL | na
| Issues addressed (#s or URLs) | na
| BC breaks? | na
| Deprecations? | na

### Required
#### Description:

This is a bug fix.

1. There's a bug where a poorly formatted or malformed phone number would cause the campaign execution to fail with an uncaught exception.

#### Steps to reproduce the bug:
1. This requires a Twilio account to test. 
2. Setup a campaign with a send sms
3. Create a contact with a bad phone number (`+55` for example)
4. Add the contact to the campaign and trigger with the `mautic:campaigns:trigger` command
5. Should get the exception

#### Steps to test this PR:
1. Apply PR 
2. Create a contact with a valid phone number and add them to the same campaign as one with a bad phone number.
3. Execute the event and notice that the event should just fail for the bad phone number but no exception is thrown.
